src/style.css

```diff
@@ -1,3 +1,113 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+
 }
+
+.header {
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+}
+
+@media (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+
+}
+@media (min-width: 1200px) {
+  .header {
+    width: 100%;
+  }
+}
```

Comment:
this task does not require fixed values using media queries, so you can delete it

Reply:
Done